### ISSUE TITLE
Mining is paying no subsidy

### ISSUE THREAD

**Issue by daedalus** [FACT0RN/FACT0RN#24]
Hi, 

It seems that the network is having some issue, currently running  

**factorn-cli getblocktemplate '{"rules": ["segwit"]}'**

gives:

**"coinbasevalue": 0**

That means no block subsidy.

I have tried this with a fresh instalation, and it happens after a full syncronization.

**Comment by daedalus** [comment]
it seems to be already solved with the latest commits.

**Comment by ReverseControl** [comment]
Hi @daedalus, this issue was addressed 5 days ago in the the last release (4.23.69, Also, see #23 ). 

All miners who mined blocks at difficulty level 257 will be paid out the coins they earned from my wallet. In addition, I will be extending this to all blocks mined at difficulty level 258 where some miners did not update their node.

As of this writing there are 4 blocks at difficulty level 259, so all 257 and 258 difficulty level blocks have been mined. For all the miners who did not waive their rights to receive coins, the final distribution is as follows:

```
fact1qrq9jz638mrxlucludwa8jj4rgg0l85au6x0dd8 :  40.21045207982
fact1qag8n79nl6f6qe35dl5uwzumghm0jk262dd6vcs :  21.93297386172
fact1qflzlapg5xzk946zwqcsngx4he08h6f65uq3vfq :  5.48324346543
Total:  67.62666940697
```

These will be paid out this week.
